### ISSUE TITLE
Additional Popover/Tooltip Placements

### ISSUE THREAD

**Issue by richaus** [kaliber5/ember-bootstrap#683]
I see that in the docs for Popover and Tooltip we have access to 4 placements (Left, Top, Bottom, & Right). Do we also have access to the additional placements available in Popper.js that BS uses? Auto, Top-End, Bottom-End, etc?

**Comment by bravo-kernel** [comment]
I was just looking for `bottom-end` myself and [found this inside the code](https://github.com/kaliber5/ember-bootstrap/blob/master/addon/components/base/bs-dropdown/menu.js#L129). 

Not quite sure how to get to the end thus far but looks do'able.

**Comment by bravo-kernel** [comment]
For the menu, popper placement works as expected. The following will lead to `bottom-end` placement:

```hbs
<dd.menu @direction="down" @align="right" as |ddm|>
  ...
</dd.menu>
```

**Comment by richaus** [comment]
Oh Fantastic! Indeed we do have 'top-end', 'bottom-end', 'top-start', 'bottom-start' etc etc. 

> I was just looking for `bottom-end` myself and [found this inside the code](https://github.com/kaliber5/ember-bootstrap/blob/master/addon/components/base/bs-dropdown/menu.js#L129).
> 
> Not quite sure how to get to the end thus far but looks do'able.

Ah fantastic! Just tried `{{#bs-popover triggerElement="#my-trigger-element" class="tooltip-inverse" placement="bottom-start"}}` and the placement was as expected 👍 👍 👍 👍 

Thanks for digging in and discovering those!

**Comment by simonihmig** [comment]
ember-bootstrap follows with its API what the original Bootstrap offers, and that only includes the 4 basic placements as mentioned. This is the public API.

The fact that popper.js supports more options, and that those options *happen to work* relies on implementation details, that are not part of the public API. I.e. in a possible future implementation change (non-breaking, i.e. minor version!) it would be possible to not use popper.js anymore, but have some other implementation that stills supports the public API, but not the other placements mentioned here.

*tl;dr* you can use those other placement options, but that's on your own risk, as they are *not* part of the public API and not guaranteed to work.
